python/faery/__main__.py

```diff
@@ -34,135 +37,58 @@ def print_help():
             # Read from file dvs.es and write to stdout
             faery input file dvs.es
 
-            # Read from stdin and write to file out.aedat4
-            faery input file dvs.es output file out.aedat4
+            # Read from UDP and write to file out.aedat4
+            faery input udp output file out.aedat4
+
+            # Display available colormaps
+            faery colormaps
```

Comment:
`colormaps` used to take an argument (the path of the output file) to make sure that we do not delete a user file by mistake. Do you override or do you ask the user if the file readily exists?

Reply:
Ahaaa
> This tells type checkers and other tools that Thing is not meant to be a private internal usage within the file but instead is meant to be a public re-export of the symbol. When the redundant import form is used, pylance will no longer report the import as unused.

Good to know, thanks!